ward/ward.lua

```diff
@@ -65,7 +70,7 @@ while i <= #arg do
       system_error("Unknown option '" .. argument .. "'")
     end
   else
-    local command = {"gcc", "-E"}
+    local command = { ccprog, "-E" }
```

Comment:
... and this part of the PR is now in master, see 4cd58de69dc938a2b609e134667afb2ecce3a5f4